WellLog.Lib/Helpers/ByteHelpers.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+
+namespace WellLog.Lib.Helpers
+{
+    public static class ByteHelpers
+    {
+        public static bool GetBit(this byte b, byte mask)
+        {
+            return (b & mask) > 0;
+        }
+
+        public static byte SetBit(this byte b, byte mask, bool value)
+        {
+            if (value) { return Convert.ToByte(b | mask); }
+            return Convert.ToByte(b & ~mask);
+        }
+
+        public static bool IsComponentRole(this byte b, byte role)
+        {
+            return (b >> 5) == (role >> 5);
+        }
```

Comment:
what about the remaining 3 bits on the other side of your shift? Does role take 3 bits?

Reply:
as per the component requirements for DLIS files, the component role resides in the highest 3 bits of the component descriptor byte.  i shift right 5 so as to ignore the lower 5 bits.

Reply:
nit: Magic numbers.  I would create a constant that's more descriptive.